mmv1/products/compute/Subnetwork.yaml

```
@@ -388,3 +395,17 @@ properties:
     output: true
     description: |
       The range of external IPv6 addresses that are owned by this subnetwork.
+  - !ruby/object:Api::Type::Boolean
+    name: 'allowSubnetCidrRoutesOverlap'
+    default_from_api: true
+    update_verb: :PATCH
+    update_url: projects/{{project}}/regions/{{region}}/subnetworks/{{name}}
+    fingerprint_name: 'fingerprint'
```

```suggestion
    update_verb: :PATCH
    update_url: projects/{{project}}/regions/{{region}}/subnetworks/{{name}}
    fingerprint_name: 'fingerprint'
```

Hey wanted to check in. Based on the previous conversation we should be able to remove `update_verb`, `update_url`, and `fingerprint_name` from this field.

---

Sorry, which previous conversation? Why would we remove them if the attribute needs these to be updated?

---

because the resource is already updatable correct? This information does not need to restated at the field level. Doing so should only be done in the case where there is either a separate endpoint to update this field or a field cannot be updated in tandem with the others.

---

Ah apologies. This resource is marked as immutable. I see now.

Do we expect more fields to be added which can be updated? I would suggest we remove immutable from the resource definition and mark other fields immutable. Lets sync up. Could you schedule a meeting with me?

---

This might be work beyond the scope of the PR though. Since this resource has already drifted off course I supposed we can accept this change. I would still like to sync though

---

Sure, I'll set up a sync!